### ISSUE TITLE
Feature Request: Post Versioning

### ISSUE THREAD

**Issue by idmontie** [Differential/meteor-blog#130]
Something similar to how Wordpress versions posts.  Wordpress has a revision history that you are able to navigate through and revert posts to previously saved versions.

I might be able to tackle this after the holidays if it's something that you would like in the main repo.

**Comment by coopermaruyama** [comment]
Yeah, I feel like maybe this should be in a separate table (e.g. `blog_post_history`) with 3 main fields: post_id, date, and post_body.

Also I'd throttle how often it creates revisions, maybe a minimum amount of time has to go by as well as a certain amount of characters added or deleted.

@hellogerard thoughts?

**Comment by dandv** [comment]
:+1: [Astronomy](https://github.com/jagi/meteor-astronomy) plans to implement support for document versioning in collections.
